dkist/wcs/models.py

```diff
@@ -130,7 +130,7 @@ def update_celestial_transform_parameters(
         -crpix[0],
         -crpix[1],
         pc,
-        transform[2].translation.value,
+        transform[2].translation.value * pc.unit if hasattr(pc, "unit") else transform[2].translation.value,
```

Comment:
What led to adding this? I think this is the slightly better way to do it though:

```suggestion
        transform[2].translation.quantity if hasattr(pc, "unit") else transform[2].translation.value,
```

Also if this is only getting hit by a benchmark it should have another unit test?